pkg/shares/share_splitting.go

```diff
@@ -126,3 +132,10 @@ func SplitBlobs(cursor int, indexes []uint32, blobs []coretypes.Blob, useShareIn
 	}
 	return writer.Export(), nil
 }
+
+func mergeMaps(mapOne, mapTwo map[coretypes.TxKey]ShareRange) map[coretypes.TxKey]ShareRange {
+	merged := make(map[coretypes.TxKey]ShareRange)
+	maps.Copy(merged, mapOne)
+	maps.Copy(merged, mapTwo)
+	return merged
```

Comment:
What happens if the two maps have common keys? which one would take precedence? I think it is worth mentioning in the function description.

Reply:
Good point. It looks like the second map's duplicate key would take precedence. Resolved in https://github.com/celestiaorg/celestia-app/pull/1276/commits/24f3d420859875283f6b31853700bd499c90c5d5